exercises/calculator.cpp

```diff
@@ -10,3 +10,42 @@
   Multiplication: 8
   Division: 2
 */
+
+
+#include<iostream>
+
+
+using namespace std;
+
+
```

Comment:
troppi spazi, levane un po'
```suggestion

#include<iostream>
using namespace std;

```

Reply:
Ho diminuito gli spazi e migliorato la leggibilità